packages/Algernon.package/ALGStaticWorldMenuType.class/class/filterMenuItems..st

```diff
@@ -0,0 +1,8 @@
+helpers
+filterMenuItems: aMorph
+	| entries |
+	entries := OrderedCollection new.
+	(aMorph isKindOf: DockingBarUpdatingMenuMorph) 
+		ifFalse: [ entries addAll: aMorph submorphs ].
+						
+	^ entries
```

Comment:
```suggestion
helpers
filterMenuItems: aMorph
	^ (aMorph isKindOf: DockingBarUpdatingMenuMorph) 
		ifTrue: [ OrderedCollection new ]
		ifFalse: [ OrderedCollection newFrom: aMorph submorphs ].
```

Reply:
Quite a nice suggestion. The current implementation seems to be similar in structure to the same method in `ALGDynamicWorldMenuType`. Should we also adjust the implementation there or is that not so relevant?

Reply:
I think so